java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -88,7 +84,7 @@ else if (osName.toLowerCase().contains("linux")) {
             if (version == null) {
```

Comment:
Hmm, ok, in that case let's remove the error and just add the following

```
return distro + "." + version.split("\\.")[0];
```

In general you don't see `String.format(...)` much in Java code.